ChaosMod/Main.cpp

```diff
@@ -25,7 +25,7 @@ void ParseConfigFile(int& effectSpawnTime, int& effectTimedDur, int& seed, int&
 
 	effectSpawnTime = configFile.ReadValueInt("NewEffectSpawnTime", 30);
 	effectTimedDur = configFile.ReadValueInt("EffectTimedDur", 90);
-	seed = configFile.ReadValueInt("Seed", 0);
+	seed = configFile.ReadValueInt("Seed", g_random.GetRandomInt(0, MAXINT));

```

Comment:
This will cause the mod to always use this seed generated by the config utility. It uses a random seed internally if the seed is set to 0 anyways (see Random.h).

Reply:
No, this is inside the ChaosMod, not the config-app. But yeah, if its already handled in random.h it should not be needed anymore 😅

Reply:
Fixed :)